### PR TITLE
Fixing an issue with the publish-plugin script

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #14

### CODE REVIEW

File: scripts/publish-plugin.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -o nounset -o errexit -o pipefail
 ROOT=$(dirname $0)/..
 WORK_PATH=$(mktemp -d)
 VERSION=$(jq -r '.version' < "${ROOT}/sdk/nodejs/bin/package.json")
-PLUGIN_PACKAGE_NAME="pulumi-resource-digitalocean-${VERSION}-$(go env GOOS)-$(go env GOARCH).tar.gz"
+PLUGIN_PACKAGE_NAME="pulumi-resource-aws-${VERSION}-$(go env GOOS)-$(go env GOARCH).tar.gz"
```
Comment (jen20) [inline scripts/publish-plugin.sh]: This looks wrong

Comment (stack72) [reply scripts/publish-plugin.sh]: great catch!

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-__NULL__
+* Fix an issue with publish-plugin.sh script that was breaking installs
```
Comment (lukehoban) [inline CHANGELOG.md]: Perhaps phrase this in user-facing terms?

Comment (stack72) [reply CHANGELOG.md]: I will reword it when I prepare for release

**Pull request merged**